src/app/file/pixly_format.cpp

```diff
@@ -0,0 +1,535 @@
+// Aseprite
+// Copyright (C) 2016  Carlo "zED" Caputo
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2 as
+// published by the Free Software Foundation.
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "app/app.h"
+#include "app/document.h"
+#include "app/file/file.h"
+#include "app/file/file_format.h"
+#include "app/file/format_options.h"
+#include "app/ini_file.h"
+#include "app/xml_document.h"
+#include "base/file_handle.h"
+#include "base/convert_to.h"
+#include "base/path.h"
+#include "doc/doc.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
```

Comment:
As a minor detail, you can `#include <cstdio>`, `<cstdlib>`, and `<cmath>` here.